docs/transaction-logging.md

```diff
@@ -0,0 +1,128 @@
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING_HOST`, host name or IP of the database, e.g., `127.0.0.1`
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING_PORT`, port on which the Postgres database is available, e.g., `5432`
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING_USERNAME`, username to log into the database
+- `CONCORDIUM_NODE_TRANSACTION_OUTCOME_LOGGING_PASSWORD`, password to log into the database
+
+As mentioned above the database must exist, otherwise the node will fail to start. If correct tables exist in the database then they will be used, otherwise the following will be executed upoon startup
+```sql
+CREATE TABLE "summaries"("id" SERIAL8  PRIMARY KEY UNIQUE,"block" BYTEA NOT NULL,"timestamp" INT8 NOT NULL,"height" INT8 NOT NULL,"summary" JSONB NOT NULL)
+CREATE TABLE "ati"("id" SERIAL8  PRIMARY KEY UNIQUE,"account" BYTEA NOT NULL,"summary" INT8 NOT NULL)
+CREATE TABLE "cti"("id" SERIAL8  PRIMARY KEY UNIQUE,"index" INT8 NOT NULL,"subindex" INT8 NOT NULL,"summary" INT8 NOT NULL)
+ALTER TABLE "ati" ADD CONSTRAINT "ati_summary_fkey" FOREIGN KEY("summary") REFERENCES "summaries"("id") ON DELETE RESTRICT  ON UPDATE RESTRICT
+ALTER TABLE "cti" ADD CONSTRAINT "cti_summary_fkey" FOREIGN KEY("summary") REFERENCES "summaries"("id") ON DELETE RESTRICT  ON UPDATE RESTRICT
+```
+
+which crates three tables, `ati`, `cti`, and `summaries`. The `ati` and `cti` stand for **a**ccount, respectively **c**ontract, **t**ransaction **i**ndex. They contain an index so that a transaction affecting a given contract or account can be quickly looked up. The outcome of each transaction is in the `summaries` table.
+
+The summary that is stored in the `summary` column of the `summaries` table is stored as a JSON value. The contents is either of the form
+```json
+{
+    "Left": ...
+}
+```
+where `...` is a transaction outcome in the same format as it appears in block summaries, or
+```json
+{
+    "Right": ...
+}
+```
+where `...` is a special transaction outcome in the same format as it appears in block summaries.
+
+## Account transaction index
+
+The meaning of the `(id, account, summary_id)` row in the `ati` table is that account `account` was affected by transaction pointed to by `summary_id`. **Affected** here means that either the account sent the transaction, or it was the target of it, for example another account sent a transfer to it. Note that accounts are stored in binary format, so as 32-byte arrays, and not in their Base58check encoding.
+
+The data is written to the table upon each finalization from oldest to newest block finalized by that round. For each block transactions are written from left to right, that is, from start to end of the block. The ids in all tables are automatically generated. Note that they should not be relied upon to be strictly sequential. Postgres does not guarantee this.
```

Comment:
I'm not sure "left to right" is helpful. Perhaps just say they're written in the order in which they occur when executing the block.

Reply:
Inded.